deps/madara/madara-runner.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+if [ -f "$RPC_API_KEY_FILE" ]; then
```

Comment:
Current logic tries to run the docker compose that is part of the submodule. You need to copy that file too and of course, some improvements can be made there cause it's structured to use Madara as FullNode

Reply:
Great. Done!